test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua

```diff
@@ -0,0 +1,140 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description: Check HMI status transitions of WEB_VIEW application
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) WebEngine App with WEB_VIEW HMI type is registered
+-- 3) WebEngine App is in some state
+--
+-- Steps:
+-- 1) WebEngine App is moving to another state by one of the events:
+--   App activation, App deactivation, Deactivation of HMI, User exit
+-- SDL does:
+--  - send OnHMIStatus notification with appropriate value of 'hmiLevel' parameter
+--
+-- Particular behavior and value depends on initial state and event, and described in 'testCases' table below
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Test Configuration ]]
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+config.checkAllValidations = true
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+--[[ Local Constants ]]
+local APP = 1
+local WEB_ENGINE = 1
+
+--[[ Local Variables ]]
+local testCases = {
+  [001] = { t = "WEB_VIEW", m = true,  s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" }
```

Comment:
Propose to change 
```suggestion
    [1] = { action = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" }
```
This relates to other scripts.

Reply:
@YarikMamykin Done (see 77f3211 )